docs/core/porting/tooling-requirements.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Tooling requirements
```

Comment:
@KathleenDollard @jamshedd when you have a chance, Marc Paine requested your feedback on this page.